examples/demo/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM mcr.microsoft.com/dotnet/sdk:6.0-jammy
 
 # install OpenTelemetry .NET Automatic Instrumentation
-ARG OTEL_VERSION=0.7.0
+ARG OTEL_VERSION=1.0.0-rc.1
```

Comment:
This is marked as "post-release' https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/releasing.md